src/Accounts/Accounts/Feedback/SendFeedback.cs

```diff
@@ -80,11 +80,6 @@ private bool CheckIfInteractive()
                     interactive = false;
                 }
             }
-
-            if (!interactive && _dataCollectionProfile != null && !_dataCollectionProfile.EnableAzureDataCollection.HasValue)
-            {
-                _dataCollectionProfile.EnableAzureDataCollection = false;
```

Comment:
It doesn't make sense for Send-AzFeedback cmdlet to update the data collection profile so I removed it.